Don't install the example files.  The second hunk is to workaround 
shells that suck at life.

http://bugs.gentoo.org/show_bug.cgi?id=89174

--- groff-1.18/doc/Makefile.sub
+++ groff-1.18/doc/Makefile.sub
@@ -60,12 +60,12 @@
 HTMLDOCIMAGEFILES=\
   pic*
 
-EXAMPLEFILES=\
+EXAMPLEFILES=#\
   webpage.ms \
   grnexmpl.g \
   grnexmpl.me
 
-PROCESSEDEXAMPLEFILES=\
+PROCESSEDEXAMPLEFILES=#\
   webpage.ps \
   gnu.eps \
   gnu.png \
@@ -170,11 +170,11 @@
 	  $(INSTALL_DATA) $$f $(docdir)/$$f; \
 	done
 	-test -d $(exampledir) || $(mkinstalldirs) $(exampledir)
-	for f in $(EXAMPLEFILES); do \
+	list="$(EXAMPLEFILES)"; for f in $${list}; do \
 	  rm -f $(exampledir)/$$f; \
 	  $(INSTALL_DATA) $(srcdir)/$$f $(exampledir)/$$f; \
 	done
-	for f in $(PROCESSEDEXAMPLEFILES); do \
+	list="$(PROCESSEDEXAMPLEFILES)"; for f in $${list}; do \
 	  rm -f $(exampledir)/$$f; \
 	  $(INSTALL_DATA) $$f $(exampledir)/$$f; \
 	done
